README.rst

```diff
@@ -48,79 +50,36 @@ through the community's forums such as mailing lists and IRC channels. Anyone
-  code from our GitHub repository.  You'll also find links for the Zephyr
-  SDK, a moderated collection of tools and libraries used to develop your
-  applications.
-
-* **Source Code in GitHub**: Zephyr Project source code is maintained on a
-  public GitHub repository at https://github.com/zephyrproject-rtos/zephyr.
-  You'll find information about getting access to the repository and how to
-  contribute to the project in this `Contribution Guide`_ document.
-
-* **Samples Code**: In addition to the kernel source code, there are also
-  many documented `Sample and Demo Code Examples`_ that can help show you
-  how to use Zephyr services and subsystems.
-
-* **Documentation**: Extensive Project technical documentation is developed
-  along with the Zephyr kernel itself, and can be found at
-  http://docs.zephyrproject.org.  Additional documentation is maintained in
-  the `Zephyr GitHub wiki`_.
-
-* **Cross-reference**: Source code cross-reference for the Zephyr
-  kernel and samples code is available at
-  https://elixir.bootlin.com/zephyr/latest/source.
-
-* **Issue Reporting and Tracking**: Requirements and Issue tracking is done in
-  the Github issues system: https://github.com/zephyrproject-rtos/zephyr/issues.
-  You can browse through the reported issues and submit issues of your own.
-
-* **Security-related Issue Reporting and Tracking**: For security-related
-  inquiries or reporting suspected security-related bugs in the Zephyr OS,
-  please send email to vulnerabilities@zephyrproject.org.  We will assess and
-  fix flaws according to our security policy outlined in the Zephyr Project
-  `Security Overview`_.
-
-  Security related issue tracking is done in JIRA.  The location of this JIRA
-  is https://zephyrprojectsec.atlassian.net.
-
-* **Mailing List**: The `Zephyr Development mailing list`_ is perhaps the most convenient
-  way to track developer discussions and to ask your own support questions to
-  the Zephyr project community.  There are also specific `Zephyr mailing list
-  subgroups`_ for announcements, builds, marketing, and Technical
-  Steering Committee notes, for example.
-  You can read through the message archives to follow
-  past posts and discussions, a good thing to do to discover more about the
-  Zephyr project.
-
-* **Chatting**: You can chat online with the Zephyr project developer
-  community and other users in two ways:
-
-  * On `Slack`_: Zephyr has dedicated channels on Slack. To register, use the
-    following `Slack Invite`_.
-
-  * IRC channel #zephyrproject on the freenode.net IRC server. You can use the
-    http://webchat.freenode.net web client or use a client-side application such
-    as pidgin (Note that all discussions have moved to Slack, although we still
-    have many developers still available on the IRC channel).
+Here's a quick summary of resources to help you find your way around:
+
+* **Zephyr Project Website**: https://zephyrproject.org
```

Comment:
Given the latest feedback, I would actually argue we could place `docs.zephyrproject.org` as the first item in this list because:

- The content is objectively better
- You are already on GitHub, so probably tech-savvy

Reply:
Sounds good, I'll move some things around to be closer to the order I imagine the average "what's in this README" beginner would want.

Reply:
Nice job